src/cow-react/modules/limitOrders/hooks/useDisableNativeTokenUsage.ts

```diff
@@ -4,10 +4,11 @@ import { limitOrdersAtom } from '@cow/modules/limitOrders/state/limitOrdersAtom'
 import { useEffect } from 'react'
 import { NATIVE_CURRENCY_BUY_TOKEN } from 'constants/index'
 import { useTradeNavigate } from '@cow/modules/trade/hooks/useTradeNavigate'
+import { getDefaultTradeState } from '@cow/modules/trade/types/TradeState'
 
 /**
  * To avoid WRAP/UNWRAP/ETH-flow cases in the Limit orders page
- * We disable selecting of native currency
+ * We disable selecting of native currency as input currency
  * We set `disableNonToken` as true to `CurrencyInputPanel` component as well
```

Comment:
This is not longer true :)

Reply:
we set it only for input currency, thanks! Will update